public/diory/diosphere.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Interesting, now I remember us talking about this diosphere.json!

In diograph-cli there's a config file called `.dcli` used for this purpose, it looks like this:
```
ffmpegPath=/opt/homebrew/bin/ffmpeg

[focus]
connectionInFocus=/tmp
roomInFocus=/tmp

[rooms./tmp]
address=/tmp
roomClientType=LocalClient
```

I wonder where this should be saved? From user's perspective who hosts this: diory-browser-electron app or the same file-storage host where also rooms are hosted?